ngrok/plan.sh

```diff
@@ -0,0 +1,77 @@
+pkg_name=ngrok
+pkg_origin=core
+pkg_version=_set_from_downloaded_zip_file_
+pkg_description="Expose local servers behind NATs and firewalls to the public internet over secure tunnels."
+pkg_upstream_url=https://ngrok.com/
+pkg_license=('proprietary')
```

Comment:
Unless I'm missing something `ngrok` is actually an apache2 license?

https://github.com/inconshreveable/ngrok/blob/master/LICENSE